RecoJets/JetProducers/src/PileupJetIdAlgo.cc

```diff
@@ -393,10 +393,19 @@ PileupJetIdentifier PileupJetIdAlgo::computeIdVariables(const reco::Jet * jet, f
 			    //To handle the electron case
 			    if(lPF!=nullptr) {
```

Comment:
Shouldn't this be considered with an "else if"  after the block started at L399?
With the present logic, if (isPacked) the d0 and dz saved here will be rewritten anyhow